### PR TITLE
Fix DD_SITE_URL in helm chart #6422

### PR DESCRIPTION
**Pull request by Gby56**
https://github.com/DefectDojo/django-DefectDojo/issues/6422

This is simply letting the user set a DD_SITE_URL in the uwsgi of django, and that will get copied over to the celery worker, I tested this with my currently deployed helm chart and it worked, Jiras finally show the link
I have set the default value explicitly, so it's visible directly and not in a yaml comment

### CODE REVIEW

**Comment by dsever** [comment]
I will take a look, but first of all create PR against of DEV not MASTER branch.
I can agree that DD_SITE_URL could be one of the main configuration parameters, but before it will be merged into dev->master you can reference the same using: extraConfig helm section.

File: helm/defectdojo/values.yaml [hunk 1]
```diff
@@ -197,6 +197,8 @@ django:
   replicas: 1
   tolerations: []
   uwsgi:
+    env:
```
Comment (dsever) [inline helm/defectdojo/values.yaml:200]: As env is in use for other components, not only for  the  django uwsgi itself, this is not natural position for given config

Comment (Gby56) [reply helm/defectdojo/values.yaml]: Done ! it seems odd to have it all the way down in the values.yaml but I understand yes, I added a comment on the default value to explain why you likely would change the value

**Comment by dsever** [comment]
So, just to explain, extraConfig is designed in the way, everything that is defined there will be created as env variable in each django based container. So if you define something like:

`extraConfig:
   DD_SOME_VAR: something
`
helm char will automatically create given env variable with assigned value. 

So it hasn't be in helm explicitly defined, you don't need to to touch helm tempates.

But if you want to specify DD_SITE_URL in extraConfig, just do it, and you don't need to change any template.

https://github.com/DefectDojo/django-DefectDojo/blob/master/helm/defectdojo/templates/configmap.yaml#L46

**Comment by Gby56** [comment]
Oh sorry right ! that should have been obvious, I forgot how extraConfigs worked, I'll remove the templates changes then, and just add an explicit DD_SITE_URL in the extraConfigs I guess ?

**Comment by Gby56** [comment]
I have added a small piece in the kubernetes readme doc then

**Review by damiencarol — APPROVED** [review]

**Review by Maffooch — APPROVED** [review]

**Pull request merged**